### PR TITLE
Merge keyword fix to main.

### PR DESCRIPTION
**Pull request by adamint**

### CODE REVIEW

**Review by phil-allen-msft — APPROVED** [review]

File: src/EFTools/EntityDesign/VisualStudio/ModelWizard/gui/WizardPageDbConfig.cs [hunk 1]
```diff
@@ -676,8 +676,17 @@ private void UpdateSettingsFromGui()
 
             // these connection strings & invariant names are coming from the ddex provider, so these are "design-time"
             var invariantName = DataConnectionUtils.GetProviderInvariantName(_dataProviderManager, _dataConnection.Provider);
+            var fixedDecryptedConnectionString = ReplaceMdsKeywords(decryptedConnectionString);
```
Comment (jasonmalinowski) [inline src/EFTools/EntityDesign/VisualStudio/ModelWizard/gui/WizardPageDbConfig.cs:679]: Wasn't this already changed in #41?

**Comment by allisonchou** [comment]
@adamint Is there anything blocking this PR? Can we get it merged in order to bring up main to rel/d17.5 & rel/d17.6?

File: src/EFTools/EntityDesign/VisualStudio/ModelWizard/gui/WizardPageDbConfig.cs (hunk 1 above)
Comment (adamint) [reply src/EFTools/EntityDesign/VisualStudio/ModelWizard/gui/WizardPageDbConfig.cs:679]: No, it was not. That was a separate instance of the same issue

File: src/EFTools/EntityDesignerVersioningFacade/ReverseEngineerDb/StoreSchemaConnectionFactory.cs [hunk 2]
```diff
@@ -136,6 +137,13 @@ internal class StoreSchemaConnectionFactory
                     targetSchemaVersion),
                 providerConnection);
         }
+        
+        private static string ReplaceMdsKeywords(string connectionString)
```
[the remark this answers is not in the document]
Comment (adamint) [reply src/EFTools/EntityDesignerVersioningFacade/ReverseEngineerDb/StoreSchemaConnectionFactory.cs:141]: We don’t have a shared assembly, and currently don’t want to risk adding

**Pull request merged**